packages/frontend/src/lib/ui/TextBox.svelte

```diff
@@ -1,10 +1,18 @@
 <script lang="ts">
+	import Button from "./Button.svelte";
+
 	export let name = '';
 	export let multiline = false;
 	export let placeholder = '';
 	export let value = '';
 	export let disabled = false;
 
+	export let copy = false;
+
+	async function copyToClipboard(){
```

Comment:
```suggestion
	async function copyToClipboard() {
```
not sure why the linter didn't pick this up